java/src/jmri/util/JmriJFrame.java

```diff
@@ -182,6 +182,19 @@ public void makePrivateWindow() {
         }
     }
 
+    /**
+     * Add this window tp the Windows Menu by adding it from the list of
```

Comment:
"tp" is misspelled. Should be "is".

It seems to me that "from" should be "to" instead.